src/modules/i18n/addZendeskTranslations.ts

```diff
@@ -0,0 +1,20 @@
+import i18next from "i18next";
+
+interface TranslationFile {
+  locale: string;
+  rtl: boolean;
+  translations: Record<string, string>;
+  updated_at: string;
+}
+
+export async function addZendeskTranslations(
```

Comment:
Maybe calling it `localize` or `loadTranslations` would be better as `addZendeskTranslations` makes it zendesk specific and maybe confusing.

Reply:
This function imports a file with a specific format, with the translation (key, value) map under `translations`. For this reason, I see it as "Zendesk-specific". 

Alternatively, we can put in the translation files just the translation map, like we are doing with the translation of the settings since we don't need the other info.

Another possible option is to remove this function and load the translations directly in each app (see next comment)